components/Banner.tsx

```diff
@@ -66,20 +74,39 @@ const Banner = ({ bannerHeight, setBannerHeight }: {
 		}
 
 		const determineSituation = (): BannerSituation => {
-            const isUnsupportedDevice = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
+			const isUnsupportedDevice = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
```

Comment:
@doomsday4 These whitespace changes are the conversion of spaces into tabs.
I suggest that you change your VSCode settings to automatically detect the indentation from the file. This file has tab-indentation throughout, except for the lines that you added.